src/qt/bitcoin.cpp

```diff
@@ -459,7 +459,7 @@ int GuiMain(int argc, char* argv[])
     SetupUIArgs();
     std::string error;
     if (!node->parseParameters(argc, argv, error)) {
-        node->initError(strprintf("Error parsing command line arguments: %s\n", error));
+        node->initError(strprintf(_("Error parsing command line arguments: %s\n"), error));
         // Create a message box, because the gui has neither been created nor has subscribed to core signals
         QMessageBox::critical(nullptr, PACKAGE_NAME,
             // message can not be translated because translations have not been initialized
```

Comment:
:eyes: Please remove the `_(` three lines above.

Reply:
Correct. Fixed.